model_compression_toolkit/common/post_training_quantization.py

```diff
@@ -178,6 +181,7 @@ def get_finalized_graph(initial_graph: Graph,
 
     Args:
         initial_graph (Graph): Graph to apply the changes to.
+        tpc (TargetPlatformCapabilities): TargetPlatformCapabilities object which holds target specific capabilities
         quant_config (QuantizationConfig): QuantizationConfig containing parameters of how the model should be
```

Comment:
Let's fix this comment

Reply:
TargetPlatformCapabilities object that describes the desired inference target platform (includes fusing patterns MCT should handle).

Reply:
Done